src/rabbit_connection_tracking_handler.erl

```diff
@@ -52,10 +52,22 @@ init([]) ->
 handle_event(#event{type = connection_created, props = Details}, State) ->
     ThisNode = node(),
     case pget(node, Details) of
-      ThisNode ->
-          rabbit_connection_tracking:register_connection(
-            rabbit_connection_tracking:tracked_connection_from_connection_created(Details)
-          );
+        ThisNode ->
+            try
+                rabbit_connection_tracking:register_connection(
```

Comment:
@michaelklishin if we're going to catch these errors, how about moving `try ... catch` into `rabbit_connection_tracking:register_connection` itself?

Reply:
But it's the handler's choice/responsibility to handle them. I don't know if we want to mask those errors for every caller and what the return value should be.

Reply:
OK